.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,11 @@
+### Feature description
+
+Describe in full detail the behavior that you would like implemented. Use screenshots or mock-ups if applicable.
```

Comment:
Can you put each sentence on its own line? It makes it easier to rearrange and delete them while also making diffs easier to read.